### ISSUE TITLE
Bought Synapse X with Crypto - Haven't received a serial key

### ISSUE THREAD

**Issue by Spaceflex** [LoukaMB/SynapseX#64]
Hey,

I also send a reply on the selly chat, but I haven't got my Synapse X key. It's been 4 days right now and I'd like a quick response/ my serial key.

My email is ruben@ooft.nl

Sicnerely,

Ruben
![Capture 2](https://user-images.githubusercontent.com/49321237/72428026-2c37eb00-378d-11ea-85cf-7e065d9f2945.PNG)
![Capture](https://user-images.githubusercontent.com/49321237/72428028-2cd08180-378d-11ea-8175-110a04bd1b38.PNG)

**Comment by Spaceflex** [comment]
10 days and still nothing.

**Comment by MX4CRZ** [comment]
I bought it with bitcoin to, have u gotten ur key yet?

**Comment by Spaceflex** [comment]
No... I've been waiting for 2 months, nobody, including 3dsboy08 doesn't respond.----- Original Message -----
